common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -814,6 +812,14 @@ div.problem {
         @include status-icon($gray-l4, "\f128");
       }
     }
+
+    &.inputtype.formulaequationinput {
```

Comment:
Yes, it probably is clearer to break this out of the nesting as `.capa_inputtype.textline.inputtype.formulaequationinput`. Will update with that.